[IMP] estate: grant full permissions to base.group_user T#71276 (#2)
ralberrto/vauxoo-training: estate/README.rst

> # Real Estate
> ---

Reviewer: This seem to be wrong. See the final result of the file

![image](https://user-images.githubusercontent.com/11479473/235735461-7bf0cce8-111b-43dd-88d0-6cec995e144d.png)

```suggestion
# Real Estate
------------------
```

> # Real Estate
> -----------------

Reviewer: ```suggestion
# Real Estate
-----------------
```

I don't know why in the file I see the - after the word estate and in the suggestion it looks good. I'm going to accept it like this for now but in the next PR please correct it, the - goes right up to the last letter of the text.

> This module is developed during the Odoo training at Vauxoo.

Reviewer: ```suggestion
This module is developed during the training of Odoo in Vauxoo.
```
